theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -296,7 +296,7 @@ def test_all(self):
                     for b in self.border_modes:
                         try:
                             self.tcase(i, f, s, db, dflip,
-                                       dprovide_shape, fd)
+                                       dprovide_shape)
```

Comment:
The fix is in the file gpuarray/tests/tests_abstract_conv.py. in the method tcase(), add the parameter fd. Raise skip test if "fd != (1, 1)".